docs/src/Table.doc.js

```diff
@@ -563,6 +920,212 @@ card(
             </Table.RowExpandable>
           </Table.Body>
         </Table>
+        </Box>
+      );
+    }
+    `}
+  />,
+);
+
+card(
+  <Example
+    name="Example: Table Row Expandable with Sticky Columns"
+    description="When specifying `stickyColumns` with expandable rows, include the column of arrows in your count. This example sets `stickyColumns` to 3."
+    defaultCode={`
+    function RowExpandableExample() {
+      const [textShown, setTextShown] = React.useState(false);
+      const showTextOnExpand = () => {
+        return <Text>Row expanded</Text>;
+      };
+
+      return(
+      <Box width={"60%"} overflow="auto">
+        <Table stickyColumns={3}>
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell>
+                <Box display="visuallyHidden">
+                  <Text weight="bold">Open/Close row</Text>
+                </Box>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Name</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Super Name</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Best Friend</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Favorite Food</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Home</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Alias</Text>
+              </Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            <Table.RowExpandable
+              accessibilityExpandLabel="Expand"
+              accessibilityCollapseLabel="Collapse"
+              id="row1"
+              onExpand={() => setTextShown(!textShown)}
+              expandedContents={
+                <Box maxWidth={236} padding={2} column={12}>
+                  <Card
+                    image={
+                      <Avatar
+                        name="tony avatar"
+                        src="https://i.ibb.co/8948ym5/avenge.png"
```

Comment:
These examples are great, but are there copyright issues with using these images? Or is this considered fair use?

Reply:
That be a good PR stunt for our gestalt open source library! :)